cc/util/file_random_access_stream.cc

```diff
@@ -49,7 +49,10 @@ int close_ignoring_eintr(int fd) {
 
 // From Google/glog
 #ifndef HAVE_PREAD
+// lseek() is not thread safe.
```

Comment:
This is needed because pread must be thread safe; a thread safe `lseek` wouldn't be enough. Consider rewording this comment.

Reply:
Yeah, maybe it is best to simply remove the comment entirely since pread() is, by design, supposed to be thread-safe?